Products/ZenUI3/browser/resources/js/zenoss/ExtOverrides.js

```diff
@@ -649,4 +649,389 @@ Ext.override(Ext.util.Sorter, {
         }
     });
 
```

Comment:
Add a comment stating what the patch is for please so we can review it when upgrading ExtJs